clang/test/Driver/fp-model.c

```diff
@@ -109,20 +112,37 @@
 // CHECK-TRAP: "-cc1"
 // CHECK-TRAP: "-ffp-exception-behavior=strict"
 
+// RUN: %clang -### -nostdinc -ffp-model=aggressive -c %s 2>&1 \
+// RUN:   | FileCheck --check-prefix=CHECK-FPM-AGGR %s
+// CHECK-FPM-AGGR: "-cc1"
+// CHECK-FPM-AGGR: "-menable-no-infs"
```

Comment:
see my other comment: pack some options (to show they are adjacent) or use `-SAME: {{^}} "-menable-no-nans"` (clang/test/Driver/linux-cross.cpp style)